### PR TITLE
Add prodigy-layer

### PR DESCRIPTION
**Pull request by CestDiego**
Manage external services from within emacs ! :D

### CODE REVIEW

File: contrib/prodigy/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - [Install](#install)
+    - [Key Bindings](#key-bindings)
+        - [Spawn prodigy](#spawn-prodigy)
+        - [Navigate through it](#navigate-through-it)
+
+<!-- markdown-toc end -->
+
+## Description
+
+This layer adds support for the [prodigy](https://github.com/rejeep/prodigy.el) package to manage external services from within Emacs (check the own package documentation for more details)
```
Comment (rphillips) [inline contrib/prodigy/README.md]: suggestion: ```check the package's documentation...```

**Comment by rphillips** [comment]
neat layer

**Comment by CestDiego** [comment]
Edited suggestions and also changed entries or entry for services/service

**Comment by CestDiego** [comment]
@syl20bnr sorry for not noticing that last comment about the screenshot section

**Comment by syl20bnr** [comment]
You know what ? I finally merged it !  :-)
I changed `SPC a r` for `SPC a S` since it is about services.

Thank you !
Cherry-picked into `develop` branch, you can safely delete your branch.

**Pull request closed without merging**